packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -84,4 +88,10 @@
     margin-left: var(--terra-menu-item-chevron-icon-margin-left, 10px);
     margin-right: var(--terra-menu-item-chevron-icon-margin-right, 10px);
   }
+
+  .instructionsforuse {
+    color: #1c1f21;
+    font-size: 12px;
+    margin-left: 10px;
```

Comment:
Lets go ahead and either make a new generic icon class for any/all icons going forward, since custom icons will be added in the future -- thinking we move lines 73-75 out from .checkmark (font-size, margin-left, margin-right) and remove 94-95 here and make a new menu item icon class so we can have single generic icon theming variables for both checkmarks and this new one (maybe something like `var (--terra-menu-item-start-icon-font-size, 1em);`)

Reply:
changed as mentioned above : f0b2838453248b6843ecae879804d04ee09854a9